### PR TITLE
Milestone 3: Style feature

### PR DESCRIPTION
**Pull request by PhillipUg**
### In this milestone, i have done the following;
- Used `flexbox` to layout Display and ButtonPanel Components
- Styled Display, ButtonPanel and Button Components
- Passed `color` and `wide` props to Button component
- Run linters and made sure everything is working properly

### CODE REVIEW

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @PhillipUg 
You have done a great job on this milestone 👏👏
However, there are some things that we need to fix.
**Kindly check my comments below**

## Status: Changes Required ♻️
_Feel free to reach out to me on slack (Fatima) if you have any questions or need any help_

File: src/components/Display.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Display = ({ result }) => (
-  <div>
+  <div className="display">
     {result}
   </div>
 );
```
Comment (fatymahmed) [inline src/components/Display.js:8]: - `Display` component should use flex display mode as mentioned on Pathwright, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -3,33 +3,33 @@ import Button from './Button';
 
 const ButtonPanel = () => (
   <div>
```
Comment (fatymahmed) [inline src/components/ButtonPanel.js]: - `ButtonPanel` component should use flex display mode as mentioned on Pathwright, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)

**Comment by PhillipUg** [comment]
### Corrected the following;
- Used flex display mode to layout ButtonPanel and Display components

**Review by jofer86 — APPROVED** [review]
## `const PROJECT_STATUS = "APPROVED"` 🚀 🚀 🚀 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All of the required changes were done, this milestone is complete!!.
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Pull request merged**